clients/src/main/java/org/apache/kafka/clients/consumer/RetriableCommitFailedException.java

```diff
@@ -22,11 +22,7 @@
 
     private static final long serialVersionUID = 1L;
 
-    public RetriableCommitFailedException(String message) {
-        super(message);
-    }
-
-    public RetriableCommitFailedException(String message, Throwable t) {
-        super(message, t);
```

Comment:
These exceptions are public classes so I don't think we can remove the existing constructors.